src/components/Map.svelte

```diff
@@ -3,16 +3,17 @@
   import 'mapbox-gl/dist/mapbox-gl.css'
   import { onMount } from 'svelte'
   import type { MapMarker } from '../types'
+  import { microcopy } from '../stores'
 
   mapboxgl.accessToken = import.meta.env.VITE_MAPBOX_PUBLIC_KEY
 
   export let map: mapboxgl.Map | null = null
   export let markers: MapMarker[] = []
-  export let lng = 10
-  export let lat = 51.3
-  export let zoom = 5.05
-  export let minZoom = 4
-  export let maxZoom = 10
+  export let lng = $microcopy?.map?.location?.lng ?? 10
```

Comment:
Nice. Maybe this should happen in `ChapterMap.svelte`?

Reply:
I would say destructure in `ChapterMap.svelte` like this

```js
const { lat, zoom, minZoom, maxZoom} =  $microcopy?.map?.location ?? {}
```

and then pass these as props to `Map`

```svelte
<Map {lat} {zoom} {minZoom} {maxZoom} />
```

Reply:
done